### PR TITLE
*: add UTF8 check global variable.

### PR DESCRIPTION
**Pull request by coocood**
Add a global variable `tidb_skip_utf8_check`, when is set to '1', do not check utf8 string.

Make adding global variable easier.

Now we don't need to upgrade bootstrap version when adding new global variables.

### CODE REVIEW

**Comment by coocood** [comment]
@shenli @zimulala PTAL

**Comment by zimulala** [comment]
Add a test case that is invalid UTF-8 for the old value.

**Comment by coocood** [comment]
@zimulala 
This is a side effect, but we don't need to make promise on it.

**Comment by zimulala** [comment]
LGTM

**Comment by coocood** [comment]
@shenli @tiancaiamao @hanfei1991  PTAL

File: executor/write.go [hunk 1]
```diff
@@ -50,13 +50,7 @@ func updateRecord(ctx context.Context, h int64, oldData, newData []types.Datum,
 			}
 			continue
 		}
-		if i < offset || i >= offset+len(cols) {
```
Comment (shenli) [inline executor/write.go]: This is use for multiple tables update. Is it safe to remove this?

Comment (coocood) [inline executor/write.go]: `assignFlag` is sliced to the single table, so we don't need `offset`.

**Comment by shenli** [comment]
LGTM

**Pull request merged**